source/backend/api/Models/Configuration/MayanConfig.cs

```diff
@@ -11,5 +11,9 @@ public class MayanConfig
         public string ConnectionPassword { get; set; }
 
         public int UploadRetries { get; set; }
+
+        public int ImageRetries { get; set; }
+
+        public int PreviewPages { get; set; }
```

Comment:
These changes are unrelated - please rebase this branch

Reply:
this has been corrected.